# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mesa/amd64-fdo5835.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Patch by Igor V. Kovalenko, with suggested change from Daniel Stone, from
freedesktop.org xorg bug #5835.

Using this patch will require that the Xorg X server is recompiled against
the new version of Mesa which includes this patch in order to get the
benefit of it.

Theoretically fixes the following bugs (untested):

https://bugs.freedesktop.org/show_bug.cgi?id=5835
https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=176976
https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=176414


--- ./src/mesa/drivers/dri/common/glcontextmodes.c.old	2005-11-30 02:02:16.000000000 +0300
+++ ./src/mesa/drivers/dri/common/glcontextmodes.c	2006-02-07 10:38:46.000000000 +0300
@@ -39,6 +39,9 @@
 # include "imports.h"
 # define __glXMemset  memset
 #else
+# if defined (HAVE_DIX_CONFIG_H)
+#  include <dix-config.h>
+# endif
 # include <X11/X.h>
 # include <GL/glx.h>
 # include "GL/glxint.h"
